zerver/lib/topic.py

```diff
@@ -141,6 +142,7 @@ def update_edit_history(
     else:
         edit_history = [edit_history_event]
     message.edit_history = orjson.dumps(edit_history).decode()
+    message.edit_history_json.insert(0, edit_history_event)
```

Comment:
This will edit the existing `edit_history_json` value. But at this stage this row may not yet have been migrated, right? And then now that it's been edited, the migration will skip it, so it'll never get backfilled.

I think instead this can just set `message.edit_history_json` to the same value `edit_history`.